tests/ast/decoder.py

```diff
@@ -444,8 +444,10 @@ def decode_timezone_expr(self, tz_expr: proto.PythonTimeZone) -> Any:
         offset_seconds = tz_expr.offset_seconds
         return timezone(offset=timedelta(seconds=offset_seconds), name=tz_name)
 
-    def binop(self, ast, fn):
```

Comment:
Is there a reason you got rid of binop over here?

Reply:
yikes!!